streaming/src/main/java/org/apache/carbondata/streaming/parser/CarbonStreamParser.java

```diff
@@ -27,9 +28,10 @@
 
   String CARBON_STREAM_PARSER = "carbon.stream.parser";
 
-  String CARBON_STREAM_PARSER_DEFAULT = "org.apache.carbondata.streaming.parser.CSVStreamParserImp";
+  String CARBON_STREAM_PARSER_DEFAULT =
+      "org.apache.carbondata.streaming.parser.RowStreamParserImp";
```

Comment:
I think it is better not to modify the default value of CARBON_STREAM_PARSER_DEFAULT, if user want to use org.apache.carbondata.streaming.parser.RowStreamParserImp, user can use it by setting `carbon.stream.parser` in the option when loading

Reply:
OK